import-export-cli/integration/devFirst_test.go

```diff
@@ -38,73 +39,76 @@ func TestInitializeProject(t *testing.T) {
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	assert.Nil(t, err, "Error while generating Project")
-	assert.Contains(t, output, "Project initialized", "TestInitializeProject Failed")
-
-	//Remove Created project and logout
-	base.RemoveDir(projectName)
-	t.Cleanup(func() {
-		base.Execute(t, "logout", apim.GetEnvName())
-	})
+	assert.Contains(t, output, "Project initialized", "Project initialization Failed")
 }
 
 //Function to initialize a project using API definition
-func InitializeAPIFromDefinition(t *testing.T, projectName string, definitionFileName string) (string, error) {
-	username := superAdminUser
-	password := superAdminPassword
-	apim := apimClients[0]
-
-	args := &initTestArgs{
-		ctlUser:   credentials{username: username, password: password},
-		srcAPIM:   apim,
-		initFlag:  projectName,
-		oasFlag:   definitionFileName,
-		forceFlag: false,
-	}
+func validateInitializeProjectWithOASFlag(t *testing.T, args *initTestArgs) {
+	t.Helper()
 
 	output, err := initProjectWithOasFlag(t, args)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	return output, err
+	assert.Nil(t, err, "Error while generating Project")
+	assert.Containsf(t, output, "Project initialized", "Test initialization Failed with --oas flag")
 }
 
 //Initialize an API from Swagger 2 Specification
 func TestInitializeAPIFromSwagger2Definition(t *testing.T) {
 	apim := apimClients[0]
 	projectName := "Swagger2APIProject"
+	username := superAdminUser
+	password := superAdminPassword
 
-	output, err := InitializeAPIFromDefinition(t, projectName, utils.TestSwagger2DefinitionPath)
-	assert.Nil(t, err, "Error while generating Project")
-	assert.Containsf(t, output, "Project initialized", "Test InitializeProjectWithDefinitionFlag Failed")
+	args := &initTestArgs{
+		ctlUser:   credentials{username: username, password: password},
+		srcAPIM:   apim,
+		initFlag:  projectName,
+		oasFlag:   utils.TestSwagger2DefinitionPath,
+		forceFlag: false,
+	}
+
+	validateInitializeProjectWithOASFlag(t, args)
 
 	//Remove Created project and logout
 	base.RemoveDir(projectName)
-	t.Cleanup(func() {
-		base.Execute(t, "logout", apim.GetEnvName())
-	})
 }
 
 //Initialize an API from OpenAPI 3 Specification
 func TestInitializeAPIFromOpenAPI3Definition(t *testing.T) {
 	apim := apimClients[0]
 	var projectName = "OpenAPI3Project"
-	output, err := InitializeAPIFromDefinition(t, projectName, utils.TestOpenAPI3DefinitionPath)
+	username := superAdminUser
+	password := superAdminPassword
 
-	assert.Nil(t, err, "Error while generating Project")
-	assert.Contains(t, output, "Project initialized", "Test InitializeProjectWithDefinitionFlag Failed")
+	args := &initTestArgs{
+		ctlUser:   credentials{username: username, password: password},
+		srcAPIM:   apim,
+		initFlag:  projectName,
+		oasFlag:   utils.TestOpenAPI3DefinitionPath,
+		forceFlag: false,
+	}
+
+	validateInitializeProjectWithOASFlag(t, args)
 
 	//Remove Created project and logout
 	base.RemoveDir(projectName)
```

Comment:
Can we move this _base.RemoveDir()_  into the _validateInitializeProjectWithOASFlag()_ function? It is an essential cleanup step when running this scenario but it is a low level detail that is not required to be known at the high level of the test case.